en/docs/catalogs/backend-jwt.md

```diff
@@ -1,14 +1,13 @@
 # BackendJWT Custom Resource
 
- This is the CR which is used to describe the backend JWT generation and it's configurations
+ This is the CR which is used to describe the backend JWT generation and it's configurations.
```

Comment:
Shall we fix this to "its configurations"?

Reply:
Fixed in PR https://github.com/wso2/docs-apk/pull/583